frontend/src/components/manageUsers/LegacySystemUsers.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import type { UserDetail } from "src/types/userTypes";
+
+import { useTranslations } from "next-intl";
+import React from "react";
+import { Table } from "@trussworks/react-uswds";
+
+export interface UsersTableProps {
+  tableDescription: string;
+  tableTitle: string;
+  users: UserDetail[];
+}
+
+function formatFullName(user: UserDetail): string {
```

Comment:
why is this different from the other implementation? can we make one general implementation and share it in a utils file?